components/Sidebar/SidebarEpisode/SidebarEpisode.styles.ts

```diff
@@ -13,15 +13,6 @@ import {
 
 export const sidebarEpisodeStyles = makeStyles((theme: Theme) =>
```

Comment:
Because no styles are using the `theme` param after these changes, remove `theme: Theme` from function params.

Reply:
Fixed!